### ISSUE TITLE
Add Civo to Packaged Kubeflow distribution

### ISSUE THREAD

**Issue by Rishit-dagli** [kubeflow/website#3336]
I recently contributed the Kubeflow distribution to [Civo Cloud](https://www.civo.com/): https://github.com/civo/kubernetes-marketplace/pull/342 . Here's a link to the Kubeflow distribution on Civo: https://www.civo.com/marketplace/Kubeflow

I would love to add this, if this sounds good?

**Comment by jbottum** [comment]
@Rishit-dagli thanks for this.   I am interested in this offer.   Will you be adding it to the list Kubeflow distributions on this page ?  https://www.kubeflow.org/docs/started/installing-kubeflow/

**Comment by Rishit-dagli** [comment]
@jbottum Yes, that is what I was intending. I will create a PR soon, please feel free to assign this issue to me

**Comment by thesuperzapper** [comment]
@Rishit-dagli if Civo is still interested in being listed on the "Installing Kubeflow" table, please raise a new PR that conforms to the new table format. 

If you need an example, here is the PR which added [deployKF](https://github.com/deployKF/deployKF) to the table: https://github.com/kubeflow/website/pull/3542

Also, as @jbottum was saying, you will only be able to list the distribution in the table, we will not be accepting new documentation per https://github.com/kubeflow/website/issues/3074.

**Comment by varodrig** [comment]
@Rishit-dagli should close this ticket or are you still pursuing this?

**Comment by thesuperzapper** [comment]
This distribution was added and renamed to QBO

/Close
